message/message.go

```diff
@@ -212,85 +218,118 @@ retry:
 	}
 
 	size := out.BX.Quad()
+
 	var buf []byte
 
-	if !c.forceLowBW && (out.CX.Low.High&MessageSatusHighBW) == MessageSatusHighBW {
-		buf = make([]byte, size)
+	if size != 0 {
+		if !c.forceLowBW && (out.CX.Low.High&MessageSatusHighBW) == MessageSatusHighBW {
```

Comment:
I have no idea :-).  I think i wanted to do it around the whole phrase.  `out  & status == status`.  But even that is unnecessary.  We just need to check if it's non-zero.  I'll rephrase this.